server/src/main/java/org/elasticsearch/transport/TransportInfo.java

```diff
@@ -77,17 +97,35 @@ public void writeTo(StreamOutput out) throws IOException {
         static final String PROFILES = "profiles";
     }
 
+    private String getPublishAddressString(String propertyName, TransportAddress publishAddress){
```

Comment:
NIT: maybe rather name this `formatPublishAddressString`, it's really weird if something is named `get...` and then used to print another object's properties :)